src/targets/gpu/compile_ops.cpp

```diff
@@ -184,8 +184,15 @@ struct compile_plan
                                    std::cout << "No binary" << std::endl;
                                return std::numeric_limits<double>::max();
                            }
-                           auto t = time_op(
-                               *ctx, cr->replace.code_object, to_shapes(cr->ins->inputs()), 20);
+                           // Time all the code objects for a given perf config and calculate total
+                           // time e.g. in case of split-K GEMM, it may or may not support fusion.
+                           // In that case MLIR compile would return code objects for individual
+                           // GEMM and pre/post fusion code objects.
+                           auto cobjs = cr->replace.code_object;
+                           double t   = std::accumulate(
```

Comment:
I think it would be better to use `transform_accumulate`(its under the `migraphx/algorithm.hpp` header).